tests/test_git.py

```diff
@@ -1569,6 +1569,58 @@ def test_sync_from_repos_without_refs(self):
 
         shutil.rmtree(new_path)
 
+    def test_rev_list(self):
+        """Test rev-list command"""
+
+        new_path = os.path.join(self.tmp_path, 'newgit')
+
+        repo = GitRepository.clone(self.git_path, new_path)
+        gitrev = repo.rev_list()
+        gitrev = [line for line in gitrev]
+
+        expected = ['456a68ee1407a77f3e804a30dff245bb6c6b872f',
+                    '51a3b654f252210572297f47597b31527c475fb8',
+                    'ce8e0b86a1e9877f42fe9453ede418519115f367',
+                    '589bb080f059834829a2a5955bebfd7c2baa110a',
+                    'c6ba8f7a1058db3e6b4bc6f1090e932b107605fb',
+                    'c0d66f92a95e31c77be08dc9d0f11a16715d1885',
+                    '7debcf8a2f57f86663809c58b5c07a398be7674c',
+                    '87783129c3f00d2c81a3a8e585eb86a47e39891a',
+                    'bc57a9209f096a130dcc5ba7089a8663f758a703',
+                    '49345fe87bd1dfad7e682f6554f7472c5576a8c7']
+
+        self.assertTrue(len(gitrev), len(expected))
+        for i in range(len(gitrev)):
+            self.assertEqual(gitrev[i], expected[i])
+
+        shutil.rmtree(new_path)
+
+    def test_rev_list_empty(self):
+        """Test if no line is returned when the log is empty"""
+
+        new_path = os.path.join(self.tmp_path, 'newgit')
+
+        repo = GitRepository.clone(self.git_path, new_path)
+        gitlog = repo.log(from_date=datetime.datetime(2020, 1, 1, 1, 1, 1))
```

Comment:
I think the test is wrong. It isn't checking the output of `rev-list`.

Reply:
sorry I forgot to delete that test